ampd/src/handlers/multisig.rs

```diff
@@ -153,7 +153,7 @@ where
             Some(pub_key) => {
                 let signature = self
                     .signer
-                    .sign(self.multisig.to_string().as_str(), msg.clone(), pub_key)
+                    .sign("axelar", msg.clone(), pub_key)
```

Comment:
Hey @ctoyan, was this deliberated ? i can return the original value if not.

Reply:
Hopefully the need for the "axelar" string is due to a fault with my setup and not with our code.

You can find more context in [this](https://eig3r.slack.com/archives/C05TRKNJDQS/p1706183731816499?thread_ts=1705932493.150129&cid=C05TRKNJDQS) slack reply.

To answer your question - it **should** work with the original version, but it doesn't work locally.

Reply:
So I'm hoping a proper ampd setup and deployment from axelar should fix it.
We can return the original value and put a TODO: or a NOTE: to which we can point the issue to the Axelar team.
WDYT?

Reply:
Hey thanks, yes it sounds good to me :+1:  Should we raise the issue on their issue tracker then ?

Reply:
@ctoyan , could you make this little change ? its hard for me to interact through forks :sweat_smile:  and probably for you is just changing and pushing a commit.

Reply:
thank you !